### DISCUSSION TITLE
How to place a logo on the server side, I can't find relevant information

### DISCUSSION THREAD

**Discussion by josetowers** [ant-media/Ant-Media-Server#6673] [General]
How to place a logo on the server side, I can't find relevant information

**Comment by yashtandon113** [comment]
Hi @josetowers 

Do you want to display the logo at the top of the Ant Media Server player (play.html) or embed it within the stream?

Which protocol (WebRTC, HLS, or DASH) are you using to play the stream?

**Comment by josetowers** [comment]
I want it to be on the server side that it is encoded, and it is hls

**Reply by yashtandon113** [reply]
@josetowers 

If you are using the Enterprise Edition of Ant Media Server, you can use the Filter Plugin. Check out [this blog post](https://antmedia.io/discover-the-filter-plugin/) for more reference. You can refer to Filter number 3 for integrating the logo in a stream.

Here is the sample API call for your reference:

**To start the filter:**

`curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "http://localhost:5080/LiveApp/rest/v2/filters/create" -d '{"filterId":"logodemo","inputStreams":["stream1"],"outputStreams":["test"],"videoFilter":"movie=/usr/local/antmedia/webapps/LiveApp/images/logo.png[logo];[in0][logo]overlay[out0]","audioFilter":"[in0]acopy[out0]","videoEnabled":"true","audioEnabled":"true","type":"asynchronous"}'`

**To Remove the filter:**

`curl -i -X DELETE -H "Accept: Application/json" -H "Content-Type: application/json" "http://localhost:5080/LiveApp/rest/v2/filters/logodemo"`


As a result, it will be as follows.

![image](https://github.com/user-attachments/assets/d63b5237-6f5e-4a01-882f-e4180f588db5)

I hope it helps. If you have any questions, please feel free to ask.

**Comment by josetowers** [comment]
I'm not interested in that version, just the community.

**Comment by josetowers** [comment]
I'm interested in putting the logo on the community version

**Comment by lastpeony** [comment]
Unless you are no using enterprise edition, only way to do that is to place the logo on client side using javascript. This way your logo will be placed on top of your video player and it will kinda look like its embedded to the stream.

But as i mentioned, on server side with community edition this is not possible.
